ansible/deploy.yml

```diff
@@ -0,0 +1,70 @@
+- name: Deploy web servers
+  hosts: webservers
+  remote_user: ec2-user
+  vars_prompt:
+  - name: secret_key
+    prompt: Enter secret key for flask session feature
+    private: true
+  - name: trello_api_key
+    prompt: Enter a trello api key generated using the steps documented in "Populating TRELLO_ Environment Variables" in the README  
+    private: true
+  - name: trello_api_token
+    prompt: Enter a trello api token generated using the steps documented in "Populating TRELLO_ Environment Variables" in the README  
+    private: true
+  - name: trello_board_id
+    prompt: Enter a trello board id found using the steps documented in "Populating TRELLO_ Environment Variables" in the README
```

Comment:
Nice, for the record I can see an argument that offering a default here would be very reasonable (since the value isn't particularly sensitive) in order to simplify the command, but it's a trade off